cmd/blockchaincmd/prompt_genesis_input.go

```diff
@@ -0,0 +1,261 @@
+		// ask in a loop so that if some condition is not met we can keep asking
+		controlAddr, cancelled, err := getAddrLoop(controllerAddrPrompt, constants.ValidatorManagerController, models.UndefinedNetwork)
+		if err != nil {
+			return nil, false, err
+		}
+		if cancelled {
+			return nil, cancelled, nil
+		}
+		if len(controlAddr) != 0 {
+			return controlAddr, false, nil
+		}
+		ux.Logger.RedXToUser("An address to control Validator Manage Contract is required before proceeding")
+	}
+}
+
+// Configure which addresses may make mint new native tokens
+func getTokenMinterAddr() ([]string, error) {
+	addTokenMinterAddrPrompt := "Currently only Validator Manager Contract can mint new native tokens"
+	ux.Logger.PrintToUser(addTokenMinterAddrPrompt)
+	yes, err := app.Prompt.CaptureNoYes("Add additional addresses that can mint new native tokens?")
+	if err != nil {
+		return nil, err
+	}
+	if !yes {
+		return nil, nil
+	}
+	addr, cancelled, err := getAddr()
+	if err != nil {
+		return nil, err
+	}
+	if cancelled {
+		return nil, nil
+	}
+	return addr, nil
+}
+
+func getAddr() ([]string, bool, error) {
+	addrPrompt := "Enter addresses that can mint new native tokens"
+	addr, cancelled, err := getAddrLoop(addrPrompt, constants.TokenMinter, models.UndefinedNetwork)
+	if err != nil {
+		return nil, false, err
+	}
+	if cancelled {
+		return nil, cancelled, nil
+	}
+	return addr, false, nil
+}
+
+func promptProofOfPossession() (string, string, error) {
+	ux.Logger.PrintToUser("Next, we need the public key and proof of possession of the node's BLS")
+	ux.Logger.PrintToUser("Check https://docs.avax.network/api-reference/info-api#infogetnodeid for instructions on calling info.getNodeID API")
+	var err error
+	txt := "What is the public key of the node's BLS?"
+	publicKey, err := app.Prompt.CaptureValidatedString(txt, prompts.ValidateHexa)
+	if err != nil {
+		return "", "", err
+	}
+	txt = "What is the proof of possession of the node's BLS?"
+	proofOfPossesion, err := app.Prompt.CaptureValidatedString(txt, prompts.ValidateHexa)
+	if err != nil {
+		return "", "", err
+	}
+	return publicKey, proofOfPossesion, nil
+}
+
+// TODO: add explain the difference for different validator management type
+func promptValidatorManagementType(
+	app *application.Avalanche,
+	sidecar *models.Sidecar,
+) error {
+	proofOfAuthorityOption := "Proof of Authority"
+	proofOfStakeOption := "Proof of Stake"
+	explainOption := "Explain the difference"
+	if createFlags.proofOfStake {
+		sidecar.ValidatorManagement = models.ValidatorManagementTypeFromString(proofOfStakeOption)
+		return nil
+	}
+	if createFlags.proofOfAuthority {
+		sidecar.ValidatorManagement = models.ValidatorManagementTypeFromString(proofOfAuthorityOption)
+		return nil
+	}
+	options := []string{proofOfAuthorityOption, proofOfStakeOption, explainOption}
+	var subnetTypeStr string
+	for {
+		option, err := app.Prompt.CaptureList(
+			"Which validator management protocol would you like to use in your blockchain?",
```

Comment:
nit: not sure if protocol is a correct word. maybe just `validator management` is good